vm/systemSmartContracts/governance.go

```diff
@@ -146,8 +146,16 @@ func (g *governanceContract) Execute(args *vmcommon.ContractCallInput) vmcommon.
 		return g.changeConfig(args)
 	case "closeProposal":
 		return g.closeProposal(args)
-	case "getVotingPower":
-		return g.getVotingPower(args)
+	case "viewVotingPower":
```

Comment:
Why did we switch from `get` to `view`? Will we do this everywhere? Otherwise let's stick with one so it would be easier for users to "guess" what endpoints could exist

Reply:
there were conflicts with internal get functions. adding view is better. but I would not change it for other contracts.